src/main/java/uk/gov/justice/probation/courtcaseservice/controller/model/ExtendedCourtCaseRequestResponse.java

```diff
@@ -45,11 +46,26 @@
     @NotEmpty
     private final List<Defendant> defendants;
 
-    public static ExtendedCourtCaseRequestResponse of(CourtCaseEntity courtCase) {
+    public static ExtendedCourtCaseRequestResponse of(CourtCaseEntity courtCase, boolean throwMultipleCourtsException) {
         return ExtendedCourtCaseRequestResponse.builder()
                 .caseNo(courtCase.getCaseNo())
                 .caseId(courtCase.getCaseId())
-                .courtCode(courtCase.getCourtCode())
+                .courtCode(Optional.ofNullable(courtCase.getHearings())
```

Comment:
Why are we keeping the courtCode at the top level here ? I don't think prepare-a-case uses this object.

Reply:
Does it not? in which case we should drop it. I thought the matcher did as well but have just checked and it doesn't. Will remove.